guppylang/compiler/expr_compiler.py

```diff
@@ -323,15 +341,13 @@ def python_value_to_hugr(v: Any, exp_ty: Type) -> val.Value | None:
                 python_value_to_hugr(elt, ty)
                 for elt, ty in zip(elts, exp_ty.element_types)
             ]
-            if any(value is None for value in vs):
-                return None
-            return val.Tuple(vs=vs)
+            if doesnt_contain_none(vs):
+                return ops.Value(ops.TupleValue(vs=vs))
```

Comment:
Mypy started complaining about this, so I added a `doesnt_contain_none` type guard